src/errors/test-run/templates.js

```diff
@@ -310,9 +310,25 @@ export default {
 
     [TEST_RUN_ERRORS.uncaughtErrorInCustomScript]: err => `
         An unhandled error occurred in the custom script:
-        
+
         Error details: ${escapeHtml(err.errMsg)}
-        
+
         ${formatExpressionMessage(err.expression, err.line, err.column)}
+    `,
+
+    [TEST_RUN_ERRORS.childWindowIsNotLoadedError]: () => `
+        Child browser window is not loaded.
+    `,
+
+    [TEST_RUN_ERRORS.childWindowNotFoundError]: () => `
+        The queried child browser window is not found.
+    `,
+
+    [TEST_RUN_ERRORS.cannotSwitchToWindowError]: () => `
+        Cannot switch to the browser window.
+    `,
+
+    [TEST_RUN_ERRORS.closeChildWindowError]: () => `
+        An unhandled error has occurred during close child windows.
```

Comment:
```js
        An error occurred while closing child windows.
```

Reply:
Dirk removed `unhandled`, should I argue?

Reply:
No, let it be. The 'An error occurred while closing child windows.' variant is normal.